scripts/events_module/relationship/romantic_events.py

```diff
@@ -672,7 +672,7 @@ def prepare_relationship_string(mate_string, cat_from, cat_to):
         """Prepares the relationship event string for display"""
         # replace mates with their names
         if "[m_c_mates]" in mate_string:
-            mate_names = [str(cat_from.fetch_cat(mate_id).name) for mate_id in cat_from.mate]
+            mate_names = [str(cat_from.fetch_cat(mate_id).name) for mate_id in cat_from.mate if not cat_from.fetch_cat(mate_id).dead and not cat_from.fetch_cat(mate_id).outside]
```

Comment:
In general ok, but the thing is, that .fetch_cat might return an None if the id is not found, which would then lead to an error because None doesn't have variables like .name or .dead or .outside.
Not sure if you can change it in this PR to prevent such errors.